docs/UserGuide.adoc

```diff
@@ -154,11 +154,18 @@ If you no longer need a note, or accidentally create a wrong one, don't worry. Y
 Format: `delete [-t] TITLE` +
 Example: `delete -t Notably/CS2103`
 
-.Demo for the `delete` command
-image:demo/Delete1.png[]
+[IMPORTANT]
+If your target path is a subtree, the whole subtree including the child would be deleted.
```

Comment:
Let's not use the term "subtree" here? I don't think people will understand what it means (a bit too technical).
Instead, we can phrase it as: "Please keep in mind that deleting a note that contains other notes will cause the whole group of notes to be deleted. You can refer to the demo below for an illustration."

Reply:
good point